CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0] - 2023-06-16
```

Comment:
As there's only a change, this "first" version can have a non-nul minor version (like `1.1.0`) so that if at some point we feel brave enough, we would be able to retroactively document a `1.0.0` version with details about all we did from, say, the point we introduced the `alpine` base rather than the `ubuntu` one.